hwstats/frontend.py

```diff
@@ -90,3 +90,13 @@ def get_time_plot_fig(pid_hash: str, session: Session, query: callable) -> Figur
     fig.add_trace(go.Scatter(x=timestamps, y=data, mode="lines"))
 
     return fig
+
+
+@app.route("/shutdown", methods=["POST"])
+def shutdown():
+    os.system("sudo shutdown -h now")
```

Comment:
@Hriz1 this looks like it's trying to issue a shutdown call to the whole system. We don't want to do that, but rather just close our database connections and exit (or shut down) the application.